modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java

```diff
@@ -277,6 +300,68 @@ private UserGroup _postUserGroup(UserGroup userGroup) throws Exception {
 		return userGroupResource.postUserGroup(userGroup);
 	}
 
+	private void _testGetUserGroupsPageWithFilters() throws Exception {
+		Page<UserGroup> page = userGroupResource.getUserGroupsPage(
+			null, null, Pagination.of(1, 10), null);
+
+		long totalCount = page.getTotalCount();
+
+		UserGroup userGroup1 = testGetUserGroupsPage_addUserGroup(
+			randomUserGroup());
+
+		UserGroup userGroup2 = testGetUserGroupsPage_addUserGroup(
+			randomUserGroup());
+
+		page = userGroupResource.getUserGroupsPage(
+			null, null, Pagination.of(1, 10), null);
+
+		Assert.assertEquals(totalCount + 2, page.getTotalCount());
+
+		userGroup1.setDescription(
+			StringUtil.toLowerCase(RandomTestUtil.randomString()));
+
+		userGroup1 = userGroupResource.patchUserGroup(
+			userGroup1.getId(), userGroup1);
+
+		Date date = userGroup1.getDateCreated();
+
+		String formattedDate = _dateFormat.format(date.getTime() - Time.SECOND);
```

Comment:
inline this but remove also the subtraction of the 1sec... and change the filters to be `lt` and the other one `ge` (greater or equal)